index.html

```diff
@@ -0,0 +1,1376 @@
+                                        class="feature img-fluid"></a>
+                            </div>
+
+                        </div>
+                        <h6 class="mt-2 font-weight-bold featured-stories-mainheadline">
+                            <a href="#" class="feature-hover">Farage Urges Johnson to Heed Trump's Call: 'Together We
+                                Would
+                                Smash Labour'</a>
+                        </h6>
+                        <p class="mt-2 featured-stories-subheadline">
+                            Brexit Party leader Nigel Farage floats an electoral pact with new Conservative U.K. Prime
+                            Minister
+                            Boris Johnson after U.S. President Donald Trump said they'd "work well" together.
+                        </p>
+                    </article>
+                    <!--/featured stories 2-->
+
+                    <!--featured stories 3-->
+
+                    <article>
+                        <div class="wrap-cover">
+                            <div class="wrap">
+                                <a href="#">U.S</a>
+                            </div>
+
+                            <div><a href="#"><img src="assets/featured-3.png" alt="featured"
+                                        class="feature img-fluid"></a>
+                            </div>
+
+                        </div>
+
+                        <h6 class="mt-2 font-weight-bold featured-stories-mainheadline">
+                            <a href="#" class="feature-hover"> Video Shows ICE Agents Smash Car Window, Drag Father Out
+                                in
+                                Front
+                                of Family</a>
+                        </h6>
+                        <p class="mt-2 featured-stories-subheadline">
+                            The family was on their way to a doctor's appointment for their daughter when their vehicle
+                            was
+                            blocked in by U.S. Immigration and Customs Enforcement agents.
+                        </p>
+                    </article>
+                    <!--/featured stories 3-->
+
+                    <!--featured stories 4-->
+
+                    <article>
+                        <div class="wrap-cover">
+                            <div class="wrap">
+                                <a href="#">Editor's Notes</a>
+                            </div>
+
+                            <div><a href="#"><img src="assets/featured-4.png" alt="featured"
+                                        class="feature img-fluid"></a>
+                            </div>
+
+                        </div>
+                        <h6 class="mt-2 font-weight-bold featured-stories-mainheadline">
+                            <a href="#" class="feature-hover"> To Our Readers:</a>
```

Comment:
Top story section is wrapped with `<section>` tag and its story inside has `<article>` tag for more semantic tags.